pandas/tests/io/json/test_normalize.py

```diff
@@ -129,6 +129,21 @@ def test_value_array_record_prefix(self):
         expected = DataFrame([[1], [2]], columns=['Prefix.0'])
         tm.assert_frame_equal(result, expected)
 
+    def test_nested_object_record_path(self):
+        # GH 22706
+        data = {'state': 'Florida',
+                'info': {
+                    'governor': 'Rick Scott',
+                    'counties': [{'name': 'Dade', 'population': 12345},
```

Comment:
From the comments in the referenced issue I don’t think we always assume this to be a list. Can you parametrize the test to be both with and without a list?